app/routes.js

```diff
@@ -17,4 +26,9 @@ exports.init = app => {
     [checkAuthentication, paramsValidator.validateSchemaAndFail(paginationSchema)],
     listUsers
   );
+  app.post(
+    '/admin/users',
+    [checkAuthentication, checkAdmin, paramsValidator.validateSchemaAndFail(signUpSchema), emailExists(true)],
```

Comment:
I would check first the schema, because if any param is invalid, checkAuthentication and checkAdmin were executed unneccesary. What do you think?